powermand/client.c

```diff
@@ -657,6 +657,30 @@ static void _telemetry_printf(int client_id, const char *fmt, ...)
     }
 }
 
+static void log_state_change(Client *c)
+{
+    char *action = NULL;
+    char *hosts;
+    char *with_errors = " with errors";
+
+    if (c->cmd->com == PM_POWER_ON)
+        action = "powered on";
+    if (c->cmd->com == PM_POWER_OFF)
+        action = "powered off";
+    if (c->cmd->com == PM_POWER_CYCLE)
+        action = "power cycled";
+    if (c->cmd->com == PM_RESET)
+        action = "reset";
+
+    if (!action)
+        return;
+
+    hosts = _xhostlist_ranged_string(c->cmd->hl);
+    syslog(LOG_DAEMON | LOG_DEBUG, "%s %s%s", action, hosts,
```

Comment:
Should be able to just use LOG_DEBUG since we already called `openlog (LOG_DAEMON)` in `daemon_init()` (right after closing file descriptors so should still be in effect).  If we set facility explicitly here then we'll have to run it down later if we change the facility or make it configurable.

Reply:
Fixed